Automattic/wp-calypso
# Implement special rules for a promo where only certain TLDs should be suggested in NUX business flow

#### Changes proposed in this Pull Request

* If one of the special promo codes is used then only suggest certain TLDs in the domain suggestion step
* Also don't check for availability for TLDs outside the limited list

#### Testing instructions

* You'll need a sandboxed API to test this
* From /start/business?coupon={one_of_the_coupons} you should only see the selected TLDs in the search results (in the sandboxed environment you'll see only one TLD)
* Also the promo code should be passed to the checkout (you'll need sandbox to test this)
